rllib/agents/dqn/dqn.py

```diff
@@ -48,6 +48,13 @@
         "final_epsilon": 0.02,
         "epsilon_timesteps": 10000,  # Timesteps over which to anneal epsilon.
     },
+
+    # Use softmax for sampling actions. Required for off policy estimation.
+    "soft_q": False,
+    # Softmax temperature. Q values are divided by this value prior to softmax.
+    # Softmax approaches argmax as the temperature drops to zero.
+    "softmax_temperature": 1.0,
```

Comment:
I would just keep the shortened name... not worth changing since the advantage is minor (and debatable).